### PR TITLE
I407 user table

### PR DESCRIPTION
**Pull request by pushyamig**

### CODE REVIEW

File: dashboard/cron.py [hunk 1]
```diff
@@ -124,26 +124,16 @@ def update_with_udw_user(self):
...
-                        where e.course_id = '{UDW_course_id}' 
-                        and e.type='StudentEnrollment' 
-                        and e.workflow_state='active' ) as e 
-                        where p.user_id=u.id 
-                        and u.id = e.user_id 
-                        and c.enrollment_id =  e.enrollment_id 
-                        and p.sis_user_id is not null
-                        """
+
+            user_sql=f"""with
```
Comment (jonespm) [inline dashboard/cron.py]: Could you fix the indentation/formatting on this SQL? It's hard to read "as-is" compared to the old format.

File: dashboard/cron.py [hunk 2]
```diff
@@ -233,9 +223,9 @@ def update_with_bq_access(self):
                                 query = '''select CAST(SUBSTR(JSON_EXTRACT_SCALAR(event, "$.object.id"), 35) AS STRING) AS file_id,
                                         SUBSTR(JSON_EXTRACT_SCALAR(event, "$.membership.member.id"), 29) AS user_id,
                                         datetime(EVENT_TIME) as access_time
-                                        FROM learning_datasets.enriched_events
+                                        FROM event_store.events
```
Comment (jonespm) [inline dashboard/cron.py]: I thought this change was already made? Maybe this is just from an old commit?

File: dashboard/cron.py (hunk 1 above)
Comment (zqian) [reply dashboard/cron.py]: +1 for the indentation improvement.

File: dashboard/cron.py (hunk 2 above)
Comment (zqian) [reply dashboard/cron.py]: +1.

File: dashboard/cron.py (hunk 1 above)
Comment (pushyamig) [reply dashboard/cron.py]: I fixed it. I have been lazy in organizing it

**Comment by pushyamig** [comment]
I fixed the latest code merge issue I have with this branch. Things are much clean now

**Pull request merged**